boards/launchxl/src/main.rs

```diff
@@ -173,11 +175,19 @@ pub unsafe fn reset_handler() {
     );
     virtual_alarm1.set_client(alarm);
 
+    cc26xx::trng::TRNG.enable();
```

Comment:
Any particular reason why this enabled by default and not upon request?

Reply:
Never thought about that, should definitely be enabled upon request :)